src/lib.rs

```diff
@@ -533,16 +535,34 @@ pub trait Rng: RngCore {
     /// let mut rng = thread_rng();
     /// println!("{}", rng.gen_bool(1.0 / 3.0));
     /// ```
+    #[inline]
+    fn gen_bool(&mut self, p: f64) -> bool {
+        assert!(0.0 <= p && p <= 1.0);
```

Comment:
As mentioned, please don't change this function in this PR. If you really think it needs changing make a new PR/issue, because it's a significant discussion point on its own.